packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -1716,10 +1717,12 @@
   "You_should_name_it_to_easily_manage_your_integrations": "You should name it to easily manage your integrations.",
   "You_will_not_be_able_to_recover": "You will not be able to recover this message!",
   "You_will_not_be_able_to_recover_file": "You will not be able to recover this file!",
+  "You_will_not_be_able_to_recover_all_files": "You will not be able to recover these files.",
   "You_wont_receive_email_notifications_because_you_have_not_verified_your_email": "You won't receive email notifications because you have not verified your email.",
   "Your_email_has_been_queued_for_sending": "Your email has been queued for sending",
   "Your_entry_has_been_deleted": "Your entry has been deleted.",
   "Your_file_has_been_deleted": "Your file has been deleted.",
+  "Your_files_are_been_deleted": "Your files are being deleted.",
```

Comment:
@danilomiranda / @gdelavald  I think the issue is that on large channels with lots of files the box will show while it's still deleting and won't go away until finished